### PR TITLE
Added Travis to Slack notification

### PR DESCRIPTION
**Pull request by frankscholten**

### CODE REVIEW

**Comment by vlal** [comment]
@frankscholten conflicts, and one comment

File: .travis.yml [hunk 1]
```diff
@@ -1,56 +1,50 @@
...
 addons:
-  ssh_known_hosts: $BASTION
+  ssh_known_hosts: "$BASTION"
 deploy:
   provider: script
   skip_cleanup: true
-  # The deploy.sh file actually points to the deploy file on the bastion. Not one in the repo.
-  script: ssh -o StrictHostKeyChecking=no $BASTION_USER@$BASTION ./deploy.sh ${REPO} $COMMIT
+  script: ssh -o StrictHostKeyChecking=no $BASTION_USER@$BASTION ./deploy.sh ${REPO}
+    $COMMIT
```
Comment (vlal) [inline .travis.yml]: Does the script still work if $COMMIT is on another line?

Comment (frankscholten) [inline .travis.yml]: Not sure. Better put it on the same line.

**Comment by pidster** [comment]
@frankscholten please resolve the conflict.

**Comment by frankscholten** [comment]
@pidster @vlal I resolved conflicts

**Pull request merged**